Sources/AST/ASTVisitor.swift

```diff
@@ -453,13 +455,22 @@ public struct ASTVisitor<Pass: ASTPass> {
   func visit(_ returnStatement: ReturnStatement, passContext: ASTPassContext) -> ASTPassResult<ReturnStatement> {
     var processResult = pass.process(returnStatement: returnStatement, passContext: passContext)
 
-    if let expression = processResult.element.expression {
-      processResult.element.expression = processResult.combining(visit(expression, passContext: processResult.passContext))
+    if let statement = processResult.element.statement {
+      processResult.element.statement = processResult.combining(visit(statement, passContext: processResult.passContext))
     }
 
     let postProcessResult = pass.postProcess(returnStatement: processResult.element, passContext: processResult.passContext)
     return ASTPassResult(element: postProcessResult.element, diagnostics: processResult.diagnostics + postProcessResult.diagnostics, passContext: postProcessResult.passContext)
   }
+  
+  func visit(_ attemptStatement: AttemptExpression, passContext: ASTPassContext) -> ASTPassResult<AttemptExpression> {
```

Comment:
Should `attemptStatement` be changed to `attemptExpression` so as to be consistent with the other places?